torchquad/tests/integration_grid_test.py

```diff
@@ -1,14 +1,53 @@
 import sys
+import pytest
+
 
 sys.path.append("../")
 
 from autoray import numpy as anp
 from autoray import to_backend_dtype
-
+import autoray as ar
 from integration.integration_grid import IntegrationGrid
+from integration.grid_integrator import GridIntegrator
+from integration.utils import _linspace_with_grads
+import tensorflow as tf
```

Comment:
I think we meant to avoid direct imports as this means you can only run things in envs where you have the individual frameworks installed. As a compromise if necessary, maybe you can wrap it in a try / except call so the test at least doesn't fail if you don't have TF in this env?

Reply:
thanks!